src/solver/tron.jl

```diff
@@ -309,9 +309,9 @@ function projected_newton!{T <: Real}(x::AbstractVector{T}, H::Union{AbstractMat
     status = stats.status
 
     # Projected line search
-    @views xfree = x[ifree]
+    xfree = x[ifree]
```

Comment:
This needs to be `@view`.

Reply:
`@views` was giving me an error, but I guess `@views` and `@view` are two different things?!

Reply:
There's possibly some change on 0.7. Using `xfree = @view x[ifree]` works.